### PR TITLE
fix pipelines for common

### PR DESCRIPTION
**Pull request by p3dr0rv**

### CODE REVIEW

**Review by rpdome — APPROVED** [review]
Stumbled into this particular error 🤣

File: azure-pipelines/templates/steps/auth-client/docker-tasks.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ steps:
     echo =============================================
     echo Run unit and instrumented inside docker container
     echo =============================================
-    docker run --privileged --cpus="3" --memory="12g" -v "$PWD":/home/gradle/ -w /home/gradle/ authclient.azurecr.io/samples/dbi-instrumented-api30 sh ../../scripts/run-instrumented-tests.sh
+    docker run --privileged --cpus="3" --memory="12g" -v "$PWD":/home/gradle/ -w /home/gradle/ authclient.azurecr.io/samples/dbi-instrumented-api30 sh scripts/run-instrumented-tests.sh
```
Comment (AdamBJohnsonx) [inline azure-pipelines/templates/steps/auth-client/docker-tasks.yml:13]: Should this be an absolute path?

Comment (p3dr0rv) [reply azure-pipelines/templates/steps/auth-client/docker-tasks.yml]: for some reason, it cant 
image

Comment (AdamBJohnsonx) [reply azure-pipelines/templates/steps/auth-client/docker-tasks.yml:13]: Wild.

**Review by AdamBJohnsonx — APPROVED** [review]

**Pull request merged**